packages/cli/templates/js/field-plugin.config.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I think this will lead to users having unexpected options (option1, option2) by default (probably without knowing it).

What if we have an empty array for `options` in the templates?

Reply:
Makes sense @eunjae-lee. 

It should be enough once we work on the documentation (EXT-1881).

Reply:
Then could you remove these options from this branch? I know it might be annoying for you to test with actual values, but they shouldn't be committed here any more.